packages/flutter_tools/bin/xcode_backend.sh

```diff
@@ -287,26 +287,24 @@ EmbedFlutterFrameworks() {
     project_path="${FLUTTER_APPLICATION_PATH}"
   fi
 
-  # Prefer the hidden .ios folder, but fallback to a visible ios folder if .ios
-  # doesn't exist.
-  local flutter_ios_out_folder="${project_path}/.ios/Flutter"
```

Comment:
Removed `flutter_ios_out_folder`.  `flutter_ios_engine_folder` initialization was moved down a few lines to where it's used.